app/services/solidus_bolt/base.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'httparty'
+
+module SolidusBolt
+  class Base
```

Comment:
We should provide a way to provide authorization headers. I think it worth do it in this PR